app/migrations/20180518_uphold.js

```diff
@@ -0,0 +1,28 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const compareVersions = require('compare-versions')
+
+// per https://github.com/brave/browser-laptop/issues/14152
+// add fingerprint exception for existing users for uphold.com
+module.exports = (data) => {
+  if (!data.lastAppVersion) {
+    return
+  }
+
+  let migrationNeeded = false
+
+  try {
+    migrationNeeded = compareVersions(data.lastAppVersion, '0.22.714') !== 1
```

Comment:
compareVersions will return -1 if less than `0.22.714` and 0 for `0.22.714`. It can only be one if greater than 714 (and in that case, it won't run)